packs/reamaze/actions/article_search.py

```diff
@@ -0,0 +1,32 @@
+from lib.actions import BaseAction
+from dotted.utils import dot
+from fuzzywuzzy import fuzz
+import json
+
+
+class ArticleSearch(BaseAction):
+    def run(self, query, include_body=False, limit=5):
+        response = self._api_get('/articles')
```

Comment:
Indeed. I've already reached out to them to try and figure this out long-term. This gets us short-term